### PR TITLE
Fixing test before passing string to normalizer;

### PR DESCRIPTION
**Pull request by seangates**
Fixed a specific use case for me with a output that is expected to be empty, but only had a newline in it. `.trim()` removes the newline, whereas just testing the string by itself it is evaluated as not falsy.

### CODE REVIEW

File: source/testers/fixtures.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ function normalizer() {
 }
 
 function cleanRenderedHtml(html) {
-  return (html ? normalizer().domString(html.trim()) : '');
+  return (html.trim() ? normalizer().domString(html.trim()) : '');
```
Comment (oxala) [inline source/testers/fixtures.js]: If you could create a use-case for that inside `test` folder, that would be appreciated.

Comment (seangates) [inline source/testers/fixtures.js]: Sure thing. Once I get to Hawaii.

Comment (oxala) [inline source/testers/fixtures.js]: Can i get to Hawaii? :-(

Comment (seangates) [inline source/testers/fixtures.js]: https://www.hawaiianairlines.com/

**Comment by oxala** [comment]
I've modified the code to extract `html.trim()` duplication. Will rebase now, thanks.

**Pull request merged**